client/src/components/Project/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { NavLink } from "react-router-dom";
+import { Button } from "antd";
+import Scrum from "./Scrums/index";
+import './style.css';
+const scrums = require('./Scrums/utilis/scrums.json')
+
+class Scrums extends Component {
+  state = {
+    project: {
+      id: '',
+      projectName: ''
+    },
+    scrums: []
+  }
+
+  componentDidMount(){
+    // fetch
+    const project = {
+      id: 1,
+      projectName: 'Demo'
+    }
+    this.setState({ scrums: scrums, project: project });
+    if(this.props.match.params !== this.state.project.projectId){
```

Comment:
this should be either inside the `render` function which is bad practice,
or inside the `componentDidUpdate` :pray: 
**because** `componentDidMount` will be only once, and it won't' be called when the component gets updated